content/terraform-docs-common/data/cloud-docs-nav-data.json

```diff
@@ -101,6 +101,19 @@
       { "title": "Explorer", "path": "api-docs/explorer" },
       { "title": "Feature Sets", "path": "api-docs/feature-sets" },
       { "title": "GitHub App Installations", "path": "api-docs/github-app-installations"},
+      {
+        "title": "Hold Your Own Key",
+        "routes": [
+          { "title": "Hold Your Own Key", "path": "api-docs/hold-your-own-key/hold-your-own-key" },
+          { "title": "Configurations", "path": "api-docs/hold-your-own-key/hyok-configurations" },
+          { "title": "Customer Key Versions", "path": "api-docs/hold-your-own-key/hyok-customer-key-versions" },
+          { "title": "Encrypted Data Keys", "path": "api-docs/hold-your-own-key/hyok-encrypted-data-keys" },
+          { "title": "Vault OIDC Configurations", "path": "api-docs/hold-your-own-key/vault-oidc-configurations" },
```

Comment:
Can we group these all in one folder called `OIDC Configurations`? Then each file below can just reference the KMS name ("AWS OIDC Configurations" can be simply "AWS")